qaframework-bdd-tests/src/test/resources/features/refapp-2.x/07-visit/visits.feature

```diff
@@ -1 +1,46 @@
-Feature: Visits management
\ No newline at end of file
+Feature: Visits management
+
+  @selenium
+  @visit
+  Scenario: Starting a visit
+    Given user logins into the system
```

Comment:
remember!, that step is only shared by the first two scenarios since for them they don't need a visited initiated by `new TestData.TestVisit(patient.uuid, TestData.getAVisitType(), getLocationUuid(homePage)).create();`. i separated them because when i combined everything like below, only one test passes (`Merge visits`)
```
    @Before(RunTest.HOOK.SELENIUM_VISIT)
    public void setUp() {
    	initiateWithLogin();
    	patient = createTestPatient();
    	new TestData.TestVisit(patient.uuid, TestData.getAVisitType(), getLocationUuid(homePage)).create();
    	new TestData.TestVisit(patient.uuid, TestData.getAVisitType(), getLocationUuid(homePage)).create();
    }
```

taking your suggestion into context, lemi fix them up and send in a commit from which you will advise